doc/api/crypto.md

```diff
@@ -1585,7 +1585,7 @@ behavior.
 
 The `algorithm` is dependent on the available algorithms supported by the
 version of OpenSSL on the platform. Examples are `'sha256'`, `'sha512'`, etc.
-On recent releases of OpenSSL, `openssl list-message-digest-algorithms` will
+On recent releases of OpenSSL, `openssl list digest-algorithms` ( `openssl list-message-digest-algorithms` for older versions of OpenSSL ) will
```

Comment:
Nit: we have a linter rule that requires wrapping long lines at 80 characters.

Reply:
Thanks for the comment. Running `make lint` didn't show any errors for me. 👍🏻

Made the change.

Reply:
Thank you. If I recall correctly, `make lint` only lints code files and code fragments in the docs, not the doc formatting.